topdoc/lib/index.js

```diff
@@ -160,10 +160,15 @@ function template(topDocument) {
           subComponent.highlightedMarkup = processMarkup(subComponent.markup);
 
           // Gather DNA data
-          subComponent.description = subComponent.description || '';
-          var subComponentDNADescription = dnaVars['spectrum-' + subComponent.id + '-description'];
-          if (subComponentDNADescription && !subComponent.ignoreDNA) {
-            subComponent.description = subComponentDNADescription + '\n\n' + subComponent.description;
+          if (subComponent.description === null) {
```

Comment:
This change lets us completely drop description/status by setting it to null. This is helpful for example pages.